main.py

```diff
@@ -0,0 +1,52 @@
+import csv
+
+
+# def read_data_for_new_task(filepath):
+#     with open(filepath, 'r') as csv_file:
+#         reader = csv.reader(csv_file)
+#         for row in reader:
+#             yield row
+
+class ReadData:
+    def __init__(self, csv_file_path):
+        while True:
+            self.count = 0
+            self.filepath = csv_file_path
+            messege = input('Введите Enter для вывода следующего значения или End для выхода из цикла: ')
+            with open(self.filepath, 'r') as csv_file:
+                reader = csv.reader(csv_file)
+                self.row_count = sum(1 for row in reader)
+                for row in reader:
```

Comment:
This is not a generator. Look through on the first part of https://pythonist.ru/generatory-python-ih-sozdanie-i-ispolzovanie/